src/chapters/spring/mvc.rst

```diff
@@ -0,0 +1,28 @@
+Design Patterns, MVC, and Spring, Oh My!
+========================================
+
+.. index:: ! design pattern, ! Model-View-Controller, ! MVC, ! model, ! view, ! controller
+
+So far, we have been designing our applications by diagrammming classes, drawing connections, and adding in interfaces.
```

Comment:
we could also say abstracting via interfaces?

Reply:
Done! Thanks!